cmake/macros/macro_deal_ii_setup_target.cmake

```diff
@@ -103,6 +103,13 @@ MACRO(DEAL_II_SETUP_TARGET _target)
     COMPILE_DEFINITIONS "${DEAL_II_USER_DEFINITIONS};${DEAL_II_USER_DEFINITIONS_${_build}}"
     )
 
+  IF (DEAL_II_MSVC)
+    SET_PROPERTY(TARGET ${_target} PROPERTY
+      RUNTIME_OUTPUT_DIRECTORY_${_build} "${CMAKE_CURRENT_BINARY_DIR}"
+      )
+  ENDIF()
```

Comment:
Ah, now I see what you try to do with that. You try to get rid of the Debug and Release folders it creates.
So, the correct location is inside ```DEAL_II_INVOKE_AUTOPILOT``` after the ```DEAL_II_SETUP_TARGET``` call.